scripts/cms-any/add-prs-to-project.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+$content = <<<EOT
+name: Add new pull request to project
```

Comment:
```suggestion
name: Add new pull requests to a github project
```
Shouldn't specify the board name though, we might change that later

Reply:
Done